node/sc/subbridge.go

```diff
@@ -210,12 +209,44 @@ func NewSubBridge(ctx *node.ServiceContext, config *SCConfig) (*SubBridge, error
 	return sc, nil
 }
 
+func (sb *SubBridge) SetRPCConn(conn net.Conn) {
+	sb.rpcConn = conn
+	go func() {
+		for {
+			data := make([]byte, rpcBufferSize)
+			rlen, err := conn.Read(data)
+			if err != nil || rlen <= 0 {
+				//logger.Error("failed to read from RPC server pipe", "err", err, "rlen", rlen)
+				time.Sleep(100 * time.Millisecond)
+				continue
+			}
+			sb.rpcSendCh <- data[:rlen]
+		}
+	}()
+}
+
+func (sb *SubBridge) SendRPCData(data []byte) error {
+	peers := sb.BridgePeerSet().peers
+	logger.Trace("send rpc message from the subbridge", "len", len(data), "peers", len(peers))
+	for _, peer := range peers {
```

Comment:
I think we need to check the RPC client can support multiple RPC server.
It it doesn't support it, we should pick a peer.

Reply:
In effect, the rpc client/server and P2P are independent modules. I think it's best to put the multi-bridge code in the next PR.